### DISCUSSION TITLE
Calling one route from within another route

### DISCUSSION THREAD

**Discussion by mckinney-bryan** [dotnet/WatsonWebserver#123] [General]
Is it possible to call one route from within another route? A example being...

ctx1 = ```xml<SomeRequest>Test</SomeRequest>```
ctx2 = ```xml<SomeNewRequest>Test2<\SomeNewRequest>```

 public static async Task Method1Rq(HttpContext ctx1) {

      // I want to pass a different xml request that is not the same as the CTX1 that was originally passed in. 
      Method2Rq(ctx2);

 }

 public static async Task Method2Rq(HttpContext ctx2) {
      await ctx.Response.Send("My Response");
 }
 
 Any help/ideas is appreciated! 

**I need to clarify my question.

**Comment by jchristn** [comment]
Hi, can you just pass HttpContextBase ctx?
